bitnami/dataplatform-bp1/values.yaml

```diff
@@ -0,0 +1,233 @@
+#   storageClass: myStorageClass
+
+## Add labels to all the deployed resources
+##
+commonLabels: {}
+
+## Add annotations to all the deployed resources
+##
+commonAnnotations: {}
+
+## Kubernetes Cluster Domain
+##
+clusterDomain: cluster.local
+
+## Extra objects to deploy (value evaluated as a template)
+##
+extraDeploy: []
+
+## Number of nodes
+##
+replicaCount: 1
+
+
+## Specifies whether RBAC resources should be created
+##
+rbac:
+  create: true
+
+## Specifies whether a ServiceAccount should be created
+##
+serviceAccount:
+  create: true
+  ## The name of the ServiceAccount to use.
+  ## If not set and create is true, a name is generated using the fullname template
+  ##
+  name:
+
+zookeeper:
+  replicaCount: 3
+  resources:
+    # We usually recommend not to specify default resources and to leave this as a conscious
+    # choice for the user. This also increases chances charts run on environments with little
+    # resources, such as Minikube. If you do want to specify resources, uncomment the following
+    # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+    limits: {}
+    #  cpu: 250m
+    #  memory: 1Gi
+    requests:
+      cpu: 250m
+      memory: 256Mi
+  affinity:
+    podAntiAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+      - labelSelector:
+          matchExpressions:
+          - key: app.kubernetes.io/name
+            operator: In
+            values:
+            - zookeeper
+        topologyKey: "kubernetes.io/hostname"
+
+## Kafka Subchart parameters
+##
+kafka:
+  enabled: true
+  replicaCount: 3
+  resources:
+    # We usually recommend not to specify default resources and to leave this as a conscious
+    # choice for the user. This also increases chances charts run on environments with little
+    # resources, such as Minikube. If you do want to specify resources, uncomment the following
+    # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+    limits: {} 
+    #  cpu: 250m
+    #  memory: 1Gi
+    requests: 
+      cpu: 250m
+      memory: 1Gi
+  affinity:
+    podAntiAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+      - labelSelector:
+          matchExpressions:
+          - key: app.kubernetes.io/name
+            operator: In
+            values:
+            - kafka
+        topologyKey: "kubernetes.io/hostname"
+    podAffinity:
+      preferredDuringSchedulingIgnoredDuringExecution:
+       - weight: 50
+         podAffinityTerm:
+           labelSelector:
+             matchExpressions:
+             - key: app.kubernetes.io/name
+               operator: In
+               values:
+                 - zookeeper
+           topologyKey: "kubernetes.io/hostname"  
+  zookeeper:
+    enabled: false
+  ## This value is only used when zookeeper.enabled is set to false
+  ##
+  externalZookeeper:
+    ## Server or list of external zookeeper servers to use.
+    ##
+    servers: dp-zookeeper
```

Comment:
I assume the intention is to use the same zookeeper instance for both Solr and Kafka, right?
In that case, I think that if we modify the Kafka chart to use tplValue with the `externalZookeeper.servers` we can provide here something like `{{ Release.name}}-zookeeper` so it does not depend on the release name.
The same could be applied to the Solr configuration.

Reply:
Sure, will do that.